test/unit/class_method_test.rb

```diff
@@ -13,7 +13,7 @@ def class_with_method(method, result = nil)
     Class.new do
       extend ClassMethods
       singleton_class.extend(ClassMethods)
-      singleton_class.define_method(method) { result } if method
+      singleton_class.send(:define_method, method) { result } if method
```

Comment:
I'd prefer that this was included in the commit in which the original change was introduced.

Reply:
Resolving - the whole branch has been re-written.